flowey/flowey_core/src/node.rs

```diff
@@ -1770,70 +1757,9 @@ pub mod steps {
             }
         }
 
-        /// Handle to a GitHub context variable.
-        ///
-        /// Includes a (non-exhaustive) list of associated constants
-        /// corresponding to global GitHub vars which are _always_ available.
-        #[derive(Serialize, Deserialize, Clone, Debug)]
-        pub struct GhContextVar {
-            is_secret: bool,
-            gh_var: Cow<'static, str>,
-        }
-
-        #[allow(non_upper_case_globals)]
-        impl GhContextVar {
-            /// `github.repository`
-            pub const GITHUB__REPOSITORY: GhContextVar = GhContextVar::new("github.repository");
-
-            /// `runner.temp`
-            pub const RUNNER__TEMP: GhContextVar = GhContextVar::new("runner.temp");
-
-            /// `github.workspace`
-            pub const GITHUB__WORKSPACE: GhContextVar = GhContextVar::new("github.workspace");
-
-            /// `github.token`
-            pub const GITHUB__TOKEN: GhContextVar = GhContextVar::new_secret("github.token");
-        }
-
-        impl GhContextVar {
-            const fn new(s: &'static str) -> Self {
-                Self {
-                    is_secret: false,
-                    gh_var: Cow::Borrowed(s),
-                }
-            }
-
-            const fn new_secret(s: &'static str) -> Self {
-                Self {
-                    is_secret: true,
-                    gh_var: Cow::Borrowed(s),
-                }
-            }
-
-            /// Get a handle to GitHub runtime variable corresponding to a
-            /// GitHub secret with the given name.
-            pub(crate) fn from_secrets(gh_var_name: impl AsRef<str>) -> Self {
-                Self {
-                    is_secret: true,
-                    gh_var: (format!("secrets.{}", gh_var_name.as_ref())).into(),
-                }
-            }
-
-            /// Check if the GitHub var is tagged as being a secret
-            pub fn is_secret(&self) -> bool {
-                self.is_secret
-            }
-
-            /// Get the raw underlying GitHub variable name
-            pub fn as_raw_var_name(&self) -> String {
-                self.gh_var.as_ref().into()
-            }
-        }
-
         #[derive(Clone, Debug)]
         pub enum GhParam<C = VarNotClaimed> {
             Static(String),
-            GhVar(GhContextVar),
```

Comment:
losing this means we are _slightly_ more inefficient when invoking certain Github Actions from flowey, as we might end up round-tripping a github native variable through a flowey Var for "no reason".

but honestly, that doesn't seem like a big deal, given the perf hit isn't particularly noticible, and could be optimized later if need be.

so this is fine